_articles/how-to-contribute.md

```diff
@@ -31,6 +31,10 @@ Contributing to open source can be a rewarding way to learn, teach, and build ex
 
 Why do people contribute to open source? Plenty of reasons!
 
+### Improve software you rely on
+
+Lots of open source contributors start by being users of software they contribute to. When you find a bug, if you are able to write a patch fixing it, you could keep it for yourself, but contributing it back is a best way to ensure that your friends (and yourself when you update to the next release) will be able to benefit from it.
```

Comment:
The second sentence is pretty long, I wonder if it could be tightened or broken up?